x/posts/internal/keeper/keeper.go

```diff
@@ -161,18 +143,9 @@ func (k Keeper) GetPostsFiltered(ctx sdk.Context, params types.QueryPostsParams)
 		var result bool
 		first, second := filteredPosts[i], filteredPosts[j]
 
-		switch params.SortBy {
-		case types.PostSortByCreationDate:
-			result = first.Created.Before(second.Created)
-			if params.SortOrder == types.PostSortOrderDescending {
-				result = first.Created.After(second.Created)
-			}
-
-		default:
-			result = first.PostID < second.PostID
-			if params.SortOrder == types.PostSortOrderDescending {
-				result = first.PostID > second.PostID
-			}
```

Comment:
I think we should revert this change as the ID sorting should be supported in any case